<launch>
    <node name="scan_values_handler" pkg="rpsexamples" type="scan_values_handler.py" output="log"/>
    <node name="driver" pkg="rpsexamples" type="driver.py" output="screen"/>
    <node name="pid" pkg="rpsexamples" type="pid.py" output="screen"/>
</launch>